<launch>
  <node pkg="ftag2test" type="testbench_controller.py" name="testbench_controller" args="" output="screen">
  </node>
  
  <include file="$(find ftag2test)/launch/logger_testbench_no_cam.launch"/>
  
  <include file="$(find ftag2)/launch/freq_test.launch"/>
</launch>
